ql/src/java/org/apache/hadoop/hive/ql/security/authorization/StorageBasedAuthorizationProvider.java

```diff
@@ -188,6 +191,17 @@ public void authorize(Table table, Privilege[] readRequiredPriv, Privilege[] wri
       checkDeletePermission(path, getConf(), authenticator.getUserName());
     }
 
+    try {
+      String user = SecurityUtils.getUGI().getShortUserName();
+      if (MetaStoreServerUtils.checkUserHasHostProxyPrivileges(user, getConf(),
+              HiveMetaStore.HMSHandler.getIPAddress())) {
+        LOG.info("Path authentication is skipped for user {} with host proxy privilege." + user);
```

Comment:
Path "authorization" not "authentication".

Reply:
done